### DISCUSSION TITLE
Online version

### DISCUSSION THREAD

**Discussion by FurblandChannel** [ThioJoe/YT-Spammer-Purge#875] [Ideas]
A public version that will allow you to submit a video/channel that will then be queued to be scanned. This could be online and would allow you to not have to set up an api key or download anything. Essentially, an online (public) front end for the program.

**Comment by Firecul** [comment]
There have been quite a few comments made on a number of discussions, the main one probably being https://github.com/ThioJoe/YT-Spammer-Purge/discussions/40

> I thought about it, and it definitely would be way easier to use, but I'd have to do a lot of research to figure out how to do that. Also the YouTube API has a relatively low initial quota and I'm not sure how hard it would be to get it increased sufficiently. I'll look into it, but don't really have any plans for it at the moment.
https://github.com/ThioJoe/YT-Spammer-Purge/discussions/40#discussioncomment-1597169

> I'll first look into just hosting the Google Cloud project, where I'm imagining the user would still run the script locally and store the credentials on their own computer. Because security-wise, I'm very hesitant to have to maintain custody of login credentials of users myself and host the actual script on my own site, so I would have to spend a lot of time making sure I do that properly.
But I think a good balance for now would be if no one had to go through the steps to set up their own cloud dashboard project and could just use one centralized one and I could request bigger quotas, but they'd still run the script locally, it would just call my API project instead of their own.
The one thing I have to look up is how they would be authorized to access the project. Right now obviously the user logs into the cloud dashboard and downloads their own client_secrets file, but I have to assume it's a different process for a public project.
https://github.com/ThioJoe/YT-Spammer-Purge/discussions/40#discussioncomment-1613972
